motioneye/config.py

```diff
@@ -406,7 +406,7 @@ def get_camera(camera_id, as_lines=False):
     camera_config = _conf_to_dict(lines,
                                   no_convert=['@network_share_name', '@network_smb_ver', '@network_server',
                                               '@network_username', '@network_password', '@storage_device',
-                                              '@upload_server', '@upload_username', '@upload_password'])
+                                              '@upload_server', '@upload_username', '@upload_password', 'camera_name'])
```

Comment:
```suggestion
                                              '@upload_server', '@upload_username', '@upload_password', '@camera_name'])
```
But let me test it quickly.

Reply:
I thought it doesn't use the '@' prefix, considering it's used like this: `data['camera_name']` while the other keys have the prefix like `data.setdefault('@upload_username', '')`

Reply:
Ah sorry, no, this is wrong, the other settings do indeed appear without `@`.